app/views/jobs/sorted.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="col-md-3"></div>
+<div class="col-md-6">
+
+<h3>Jobs by Location</h3>
+
+<% @sorted.keys.each do |location| %>
+  <h4><%= location %>:</h4>
+  <ul>
+    <% Job.where(city: location).each do |job| %>
```

Comment:
this is again a lot of logic in your views - we don't want to query our db in our views. that's the responsibility of our controller.